registry/handlers/referrers.go

```diff
@@ -195,6 +229,19 @@ func readlink(ctx context.Context, path string, stDriver driver.StorageDriver) (
 	return digest.Parse(string(content))
 }
 
+func generateLinkHeader(repoName, subjectDigest, artifactType string, pageSize int, p int) string {
```

Comment:
nit: code style: variable naming
```suggestion
func generateLinkHeader(repoName, subjectDigest, artifactType string, pageSize int, pageNumber int) string {
```